build/tasks/typescript.js

```diff
@@ -124,31 +124,31 @@ gulp.task('typescript:app', function () {
 var clarityHtmlFiles = ["src/clarity-angular/**/*.html"];
 gulp.task('typescript:clarity:watch', function () {
     gulp.watch(claritySources.concat(clarityHtmlFiles), function () {
-        return runSequence('tslint:clarity:no-error', 'typescript:clarity');
+        return runSequence('tslint:clarity', 'typescript:clarity');
```

Comment:
No, linting should not be part of the build anymore, at all. It should be a separate task, like formatting, that only runs when required or as part as an overall check on Travis.